docs/CRSF.md

```diff
@@ -40,36 +45,35 @@ Configuration of MAVLink stream rates
 - SRx_POSITION = 2
 - SRx_RAW_SENS = 0 (for most of you this one is unimportant, keep it at 0 unless you really need it)
 
-Configuration for CRSF receiver
+Configuration for CRSF Receiver:
 
-Setting up ArduPilot for a CRSF receiver can be a bit tricky, as it can depend on the used board, and might need BRD_ALT_CONFIG to be set to a specific value. It is best to consult the ArduPilot wiki, or ask in the ArduPilot discussion channel.
+Setting up ArduPilot for a CRSF receiver can be a bit tricky, as it depends on the flight controller board, and might need BRD_ALT_CONFIG to be set to a specific value. It is best to consult the ArduPilot wiki, or ask in the ArduPilot discussion channel.
 
 For my Matek H743 board the configuration is:
 
 - BRD_ALT_CONFIG = 1
 - RC_PROTOCOLS = 536 or 512
-- SERIAL7_BAUD = irrelevant
+- RSSI_TYPE = 3
```

Comment:
no, not for everyone, I e.g. have 5!
so, either drop it, add a note, or describe the cases ...
it's not functional for the working of teh crsf setup, so I'm not sure it is required and thus would rather drop it, or you add a whole subchapter on it
rssi is a real tricky thing!!

Reply:
Moved RSSI_TYPE to notes and recommended 3 or 5.